### PR TITLE
Add 'sel gris' ingredient

### PR DESCRIPTION
**Pull request by raphael0202**
Update ingredient taxonomy.
Add 'sel gris marin' and 'sel gris de Guérande' ingredient.

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -13499,6 +13499,19 @@ de:Salz aus der Camargue
 fr:Sel de camargue
 nl:Zout uit de Camargue
 
+<en:sea salt
+en:sel gris, grey salt
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:13503]:
```suggestion
en:grey salt, sel gris
```

Are you sure "gray salt" is always sea salt?

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -13499,6 +13499,19 @@ de:Salz aus der Camargue
...
+fr:sel gris, sel marin gris, sel gris marin
+wikipedia:en:https://en.wikipedia.org/wiki/Sel_gris
+wikidata:en:Q5493196
+# ingredient/fr:sel-gris-marin has 32 products in french @2020-04-03
+
+<en:sel gris
+<en:Salt from Guérande
+fr:sel gris de Guérande
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:13511]: Let's add the English name. 
We have 3 products https://world.openfoodfacts.org/ingredient/grey-salt-from-guerande

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (raphael0202) [reply taxonomies/ingredients.txt:13503]: "Sel gris" is the real term in english (https://en.wikipedia.org/wiki/Sel_gris), grey salt is a term I've found on the wikipedia and wikidata page. I'm ok with deleting "grey salt" though.

Comment (VaiTon) [reply taxonomies/ingredients.txt:13503]: > Sel gris (pl. sels gris, "gray salt" in French) is a coarse granular sea salt

From Wikipedia

Comment (raphael0202) [reply taxonomies/ingredients.txt:13503]: It seems it's always sea salt: https://oureverydaylife.com/what-is-gray-salt-12234109.html

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (raphael0202) [reply taxonomies/ingredients.txt:13511]: Added.

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (stephanegigandet) [reply taxonomies/ingredients.txt:13503]: We should put the term that is most common on ingredients list. I have not found "sel gris" on any English ingredient list. But we do have some instances of "grey salt".

grey-salt-from-guerande | 3 | *
-- | -- | --
island-of-re-grey-sea-salt | 1 | *
grey-sea-salt | 1 | *
fr:thefine-table-salt-is-made-from-dried-grey-salt | 1 | *
fr:french-grey-sea-salt | 1 | *
fr:coarse-grey-salt-is-harvested-every-day-in-summerfrom-the-clay-beds-of-the-salt-marshes | 1 | *

Wikipedia is not always right. :)

**Pull request merged**